src/main/cpp/t04_swap.cpp

```diff
@@ -25,6 +25,42 @@
 using namespace std;
 
 int t04_swap() {
-    
+	int a[100][100];
+	int n;
+	int m;
+	cin >> n >> m;
+	int p, q;
+	for (int i = 0; i < n; i++)
+	{
+		for (int j = 0; j < m; j++)
+		{
+			cin >> a[i][j];
+		}
+
+	}
+	cin >> p >> q;
+	
+	int boof=0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'boof' can be reduced.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4951809)